openupgradelib/openupgrade.py

```diff
@@ -679,12 +679,15 @@ def rename_fields(env, field_spec, no_deep=False):
                 AND context ~ %(old_pattern)s
             """ % {
                 'old_pattern': (
-                    r"$$('group_by'|'col_group_by'):([^\]]*)"
+                    r"""$$('group_by'|'col_group_by'|'graph_groupbys'|'graph_measure'
```

Comment:
`graph_measure` passes a single field, so it doesn't work in this regexp.

Reply:
You' right, I correct it, please re-review